3D Chess/Assets/Scripts/Piece.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class Piece : MonoBehaviour
+{
+    public PieceType type = default;
+
+    private Events events = default;
+
+    public void Select()
+    {
+        gameObject.GetComponent<Renderer>().material.color = new Color(1, 0, 0, 1);
```

Comment:
[nit] Extracting these colors to constants will make things a little more readable.

Reply:
Extracted with 4233589.